### PR TITLE
PAR-1114

### PR DESCRIPTION
**Pull request by jon-edery**

### CODE REVIEW

File: sync/par_flows.par_flow.send_enforcement.yml [hunk 1]
```diff
@@ -3,13 +3,14 @@ langcode: en
...
 _core:
-  default_config_hash: Y0uOwoGFUvq3k-KLidK-FHckxL3SIT2NseFp8Zrk-NE
+  default_config_hash: HWIlwz4Bkq7Rjw7b2Zxj4uVj2dgr6ZW8bBkbXr-jR58
 id: send_enforcement
 label: 'Send Enforcement'
 default_title: null
 default_section_title: null
 description: 'The send operations for an enforcement notice.'
 save_method: end
+states: null
```
Comment (kalpaitch) [inline sync/par_flows.par_flow.send_enforcement.yml]: Actually, something you're probably not aware of, this should probably be:
```
states:
  - par_data_enforcement_notice
```

Comment (jon-edery) [reply sync/par_flows.par_flow.send_enforcement.yml]: updated

**Comment by stemount** [comment]
`drush fu par_enforcement_flows`

**Comment by jon-edery** [comment]
Tests broken in master which result in the build failing Paul is updating front end test to fix this the fail is unrelated to this code so merging this so it can be testing further.

**Pull request merged**